mycore-base/src/main/resources/config/messages_de.properties

```diff
@@ -227,8 +227,8 @@ metaData.versions.action.M = ge\u00E4ndert
 
 mycore.version = MyCoRe Version: {0}
 
-object.delObject   = L\u00F6schen dieses Dokumentes
-object.editObject  = Bearbeiten dieses Dokumentes
+object.delObject   = L\u00F6schen dieses Dokuments
+object.editObject  = Bearbeiten dieses Dokuments
 object.hasChildren = Dieses Objekt hat Kinder
```

Comment:
Warum spricht man hier manchmal vom Objekt und manchmal vom Dokument?

Reply:
Also für mich ist das falsch so.
https://deutsch.heute-lernen.de/grammatik/der-die-das/dokument/deklination

Reply:
In meinem Sprachgebrauch bin ich auch eher beim ursprünglichen "des Dokumentes".
Man sollte nur nochmal schauen, das wir das zumindest MyCoRe/MIR/UBO intern einheitlich handhaben.